packages/frontend/public/locales/ua/common.json

```diff
@@ -74,7 +74,11 @@
       "btn": {
         "auction": "Аукціон",
         "or": "або",
-        "directSale": "Продаж"
+        "directSale": "Продаж",
+        "tooltip": {
+          "auction": "Аукціон",
+          "directSale": "Продаж"
```

Comment:
your tooltip should indicate why the button is disabled.

You need to verify you phone number ....

Reply:
done